src/request.js

```diff
@@ -12,6 +13,9 @@ import {
 import nameFor from './naming';
 
 
+const DEFAULT_TIMEOUT = 5000;
```

Comment:
We don't have anything like `@default` in nodule?

Reply:
Nope - but seems like great idea for https://github.com/globality-corp/nodule-config